artemis/values.yaml

```diff
@@ -102,7 +102,7 @@ artemis:
       # kubernetes.io/ingress.class: nginx
       # kubernetes.io/tls-acme: "true"
     hosts:
-      - host: example.com 
```

Comment:
Just read that `example.com` is also not registered to anyone but I guess `artemis.example` is nicer for reading ;) and there are no conflicts.